tests/infra/basicperf.py

```diff
@@ -383,72 +406,40 @@ def cli_args():
         help="List of hostnames[,pub_hostnames:ports]. If empty, spawn minimum working number of local nodes (minimum depends on consensus and other args)",
         action="append",
     )
-    client_args_group = parser.add_mutually_exclusive_group()
-    client_args_group.add_argument(
-        "-cn",
-        "--client-nodes",
-        help="List of hostnames for spawning client(s). If empty, one client is spawned locally",
-        action="append",
-    )
-    client_args_group.add_argument(
-        "--one-client-per-backup",
-        help="If set, allocates one (local) client per backup",
-        action="store_true",
-    )
-    parser.add_argument(
-        "-nlc",
-        "--num-localhost-clients",
-        help="The number of localhost clients. \
-        This argument is cumulative with the client-nodes and one-client-per-backup and arguments",
-    )
-    parser.add_argument(
-        "--send-tx-to",
-        choices=["primary", "backups", "all"],
-        default="all",
-        help="Send client requests only to primary, only to backups, or to all nodes",
-    )
     parser.add_argument(
         "--use-jwt",
         help="Use JWT with a temporary issuer as authentication method.",
         action="store_true",
     )
-    parser.add_argument(
-        "--repetitions",
-        help="Number of requests to send",
-        type=int,
-        default=100,
-    )
-    parser.add_argument(
-        "--write-tx-times",
-        help="Unused, swallowed for compatibility with old args",
-        action="store_true",
-    )
     parser.add_argument(
         "--client-timeout-s",
         help="Number of seconds after which unresponsive clients are shut down",
         default=90,
         type=float,
     )
-    parser.add_argument(
-        "--rw-mix",
-        help="Run a batched, fractional read/write mix instead of pure writes",
-        type=float,
-    )
     parser.add_argument(
         "--max-writes-ahead",
         help="Maximum number of writes to send to the server without waiting for a response",
         type=int,
         default=1000,
     )
-
+    parser.add_argument(
+        "--key-space-size",
+        help="Size of the key space to be pre-populated and which writes and reads will be performed on",
+        type=int,
+        default=100,
```

Comment:
I think this is reducing the key-space in existing runs? Looks like `RWMix` was using 1000 at least, although `write_to_random_keys` was 100. I think that's fine, and 100 is a sensible default, just noting here in case it causes unexpected shift.

Reply:
Yes, this is done to shorten the pre-launch, which is currently slower because it's using the Python client. Once that's done through either a custom endpoint or faster somehow, we can increase the key range again.